src/views/Home.vue

```diff
@@ -1,36 +1,55 @@
 <template>
   <header
     id="home"
-    :style="{ 'background-image': headerImage }"
+    class="cover-image cover nested-img vh-100 bg-center center"
   >
-    <div class="logo">
+    <NavBar />
+    <div class="pt8 ma3 center measure-wide">
       <img
-        :src="headerLogo"
+        src="../assets/vector/bytie-gradient.svg"
         alt="BYTIE illustration logo"
+        class="img "
       >
-      <h1>ILLUSTRATIONS</h1>
+      <h1 class="near-white center tc tracked-wide ttu f3 ph4">
+        Illustrations
+      </h1>
     </div>
-    <HelloWorld />
+    <!--    <HelloWorld />-->
   </header>
 </template>
 
 <script>
 // @ is an alias to /src
-import HelloWorld from '@/components/HelloWorld.vue'
+/* import HelloWorld from '@/components/HelloWorld.vue' */
+import NavBar from '../components/NavBar'
```

Comment:
it's recommended to use `@` for components that don't rely on the folder structure

`import NavBar from '@/components/NavBar'` (might not work because of the missing .vue)
`import NavBar from '@/components/NavBar.vue'`